packages/neon/neon/lib/src/pages/home.dart

```diff
@@ -176,6 +175,7 @@ class _HomePageState extends State<HomePage> {
 
         return body;
       },
+      child: appView,
```

Comment:
Why is this needed/why was it not done before?

Reply:
The child parameter let's you cache the subtree but we alreeady cache it manaually in the appView variable.

There is no difference to before and I can see how the original way is easier to understand.